tox.ini

```diff
@@ -26,7 +26,7 @@ deps =
        moto==4.0.3
        s3fs==0.4.2
 commands =
-       pytest -n {posargs} -s -v --timeout=300 --reruns=2 --reruns-delay=15 \
```

Comment:
I have removed this for testing purposes only so we should put it back

Reply:
Reverted, thanks